### ISSUE TITLE
[Feature Request] Selected architectures for Android .apk

### ISSUE THREAD

**Issue by itlancer** [airsdk/Adobe-Runtime-Support#2873]
## Feature Description
AIR should support to specify architectures that will be included to .apk build.
If you distribute Android application with private stores or EMM/MDM systems - then in most cases you should provide APK, not AAB. And if you need to support multiple architectures you should provide APK with multiple architectures. But sometimes including "all" four architectures redundantly and cause APK extra file size that should be avoided.

Right now you can just specify only one architecture for APK or "all": https://airsdk.dev/docs/building/air-developer-tool/commands/package#-arch
But there is no option to build just armv7+armv8 APK, for example.

May be it could be implemented via ADT `-package` `-arch` comma separated values support (`armv7,armv8` for example).
Or `<buildArchitectures>armv7,armv8</buildArchitectures>` support in application manifest for APK https://airsdk.dev/docs/building/application-descriptor-files/elements/android#buildarchitectures. Right now it works only for AAB, not APK.

Related issue: https://github.com/airsdk/Adobe-Runtime-Support/issues/2009

## Known Workarounds
1) https://github.com/airsdk/Adobe-Runtime-Support/issues/2009#issuecomment-1174606337
Build with `-target android-studio`, modify build.gradle (Module: application-name.app) (which is actually just in the project folder, subfolder "app", filename "build.gradle").
by adding something a 'ndk'/'splits' sections:

```
android {
    some stuff...
    defaultConfig {
      other stuff...
      ndk {
            abiFilters "armeabi-v7a", "arm64-v8a"
        }
    }
    compileOptions {
        more stuff...
    }
    splits {
        abi {
            enable true
            reset()
            include "armeabi-v7a", "arm64-v8a"
            universalApk true
        }
    }
}
dependencies {
...
```
and build project (APK) via Android Studio.
2) https://github.com/airsdk/Adobe-Runtime-Support/issues/2009#issuecomment-1693080471
Build AAB with selected architectures and then convert it to APK.

**Comment by itlancer** [comment]
@ajwfrost 
Now it possible using AIR SDK 50.2.3.7 and specify, for example,
`<buildArchitectures>armv7,armv8</buildArchitectures>`
in application XML. Thank you!
But if you AIR application for Android using native extension with native code (.so files) for all platforms (armv7, armv8, x86, x86_64) then in output .APK file all such .so files will be included. Not just armv7 and armv8.
I mean that .APK will have:
```
MyApp.apk/lib/arm64-v8a/libMyNativeExtension.so
MyApp.apk/lib/armeabi-v7a/libMyNativeExtension.so
MyApp.apk/lib/x86/libMyNativeExtension.so
MyApp.apk/lib/x86_64/libMyNativeExtension.so
```
not just 
```
MyApp.apk/lib/arm64-v8a/libMyNativeExtension.so
MyApp.apk/lib/armeabi-v7a/libMyNativeExtension.so
```

**Comment by itlancer** [comment]
Fixed with latest AIR SDK 50.2.3.8.
Thanks!
